scripts/perf_run.py

```diff
@@ -0,0 +1,137 @@
+"""
```

Comment:
Maybe better to rewrite this as ipython notebook (the main advantage - you can easily add tables/visualize your result)
CC: @piskvorky

Reply:
I agree. Like this, with hardwired paths to `C:/rare/corpus...`, the script is not very useful.